src/utils/transformer.ts

```diff
@@ -15,9 +15,9 @@ export const toEntities = (variant: RegVariant, dirs: Dirs, items: RegItem[]): R
       id,
       variant,
       name: item.raw,
-      diff: path.join(dirs.diff, item.raw),
-      before: path.join(dirs.expected, item.raw),
-      after: path.join(dirs.actual, item.raw),
+      diff: path.join(dirs.diff, item.encoded),
+      before: path.join(dirs.expected, item.encoded),
+      after: path.join(dirs.actual, item.encoded),
```

Comment:
> If this difference is intentional, this PR may break something.
> I am grad to give me some comments on this.

Don't worry, this is my mistake :innocent:
It will be perfect work :+1: